test/general/TestFill.py

```diff
@@ -20,8 +25,12 @@ def generate_multi_world(players: int = 1) -> MultiWorld:
                         "Menu Region Hint", player_id, multi_world)
         multi_world.regions.append(region)
 
+        for option_key in itertools.chain(Options.common_options, Options.per_game_common_options):
+            option_value = getattr(args, option_key, {})
+            setattr(multi_world, option_key, option_value)
+            multi_world.worlds[player_id].options[option_key] = option_value[player_id]
+
```

Comment:
Shouldn't `world.set_options(args)` correctly set up the world with args coming from `default_common_options`?

Reply:
The worlds get instantiated in `set_options` but `TestFill` builds the worlds so I can't just call `set_options` in it. `TestFill` is a bit janky tbh